pom.xml

```diff
@@ -102,24 +102,45 @@
       <artifactId>devon4j-basic</artifactId>
       <version>${devon4j.version}</version>
     </dependency>
+    
     <dependency>
       <groupId>com.devonfw.java.modules</groupId>
       <artifactId>devon4j-test</artifactId>
       <version>${devon4j.version}</version>
       <scope>test</scope>
     </dependency>
+    
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
       <version>1.2.0</version>
       <scope>test</scope>
     </dependency>
+    
     <dependency>
       <groupId>org.codehaus.sonar-plugins.java</groupId>
       <artifactId>java-checks</artifactId>
       <version>3.2</version>
       <scope>test</scope>
     </dependency>
+    
+	  <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+	  </dependency>
+    
+	  <dependency>
+      <groupId>net.sf.m-m-m</groupId>
+      <artifactId>mmm-util-core</artifactId>
+      <version>8.7.0</version>
+	  </dependency>
```

Comment:
Can you format this properly and fix indendation please.

Reply:
I'm at a loss at what is going wrong here... pom.xml is formatted just fine when I open it locally (in devon-ide or vscode). Maybe github changes the indentation for some reason?
![pom_indentation](https://user-images.githubusercontent.com/49903872/70625903-b12a6480-1c23-11ea-99da-38e91de78a61.PNG)

Reply:
I think it's fine now... Problem was that I indented with 1 tab which relates to 4 spaces in my ide, but 6 or 8 spaces for GitHub.